.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,18 +1,24 @@
 ---
-name: "\U0001F41B Bug 报告"
-about: 错别字、语法错误等。
+name: "\U0001F41B Bug Report"
+about: Typo, Grammatical error, etc.。
 
 ---
 
 ## Bug 报告
 
-**本仓库仅用于解决与中文文档相关的问题，其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/)**
+**This repository is ONLY used to solve problems related to DOCS-CN.
+PLease move other questions(TiDB, TiKV, etc) to [other repositories](https://github.com/pingcap/).**
```

Comment:
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**